uber/templates/jobs/form.html

```diff
@@ -137,16 +137,14 @@ <h2>
       </select>
     </div>
   </div>
-  {% if job.type == c.REGULAR %}
-    <div class="form-group">
-      <label class="col-sm-3 control-label">Extra 15{{ macros.popup_link("../static_views/extra15.html") }}</label>
-      <div class="col-sm-6">
-        <div class="form-control-static checkbox">
-          {{ macros.checkbox(job, 'extra15', label='Yes') }}
-        </div>
+  <div class="form-group">
```

Comment:
I don't think we need to allow the 'extra 15' property for setup/teardown shifts? They're meant to be simply large blocks of time of availability, right?

Reply:
I guess not? I asked jkettle if he wanted to be able to add the extra 15, and he said it wouldn't hurt. I figured other departments might also want it, like for setting up reg, etc...

This might be a bigger policy change than I thought. In which case I think it would be fine to revert the "extra15" changes.